src/wordpress-importer.php

```diff
@@ -1,1236 +1,65 @@
 <?php
-/*
-Plugin Name: WordPress Importer
-Plugin URI: https://wordpress.org/plugins/wordpress-importer/
-Description: Import posts, pages, comments, custom fields, categories, tags and more from a WordPress export file.
-Author: wordpressdotorg
-Author URI: https://wordpress.org/
-Version: 0.6.5-alpha
-Text Domain: wordpress-importer
-License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
-*/
-
-if ( ! defined( 'WP_LOAD_IMPORTERS' ) )
-	return;
-
-/** Display verbose errors */
-define( 'IMPORT_DEBUG', false );
-
-// Load Importer API
-require_once ABSPATH . 'wp-admin/includes/import.php';
-
-if ( ! class_exists( 'WP_Importer' ) ) {
-	$class_wp_importer = ABSPATH . 'wp-admin/includes/class-wp-importer.php';
-	if ( file_exists( $class_wp_importer ) )
-		require $class_wp_importer;
-}
-
-// include WXR file parsers
-require dirname( __FILE__ ) . '/parsers.php';
-
 /**
- * WordPress Importer class for managing the import process of a WXR file
+ * WordPress Importer Git loader
```

Comment:
One more - looks like this file description got incorrectly moved from the root `wordpress-importer.php` file to this one.